src/index.html

```diff
@@ -11,21 +11,18 @@ <h1>Leaderboard</h1>
       <section class="display-container">
         <header class="display-heading">
           <h2>Recent Scores</h2>
-          <button type="submit">Refresh</button>
+          <button id="refresh-button" type="submit">Refresh</button>
         </header>
         <div class="display-board">
```

Comment:
- [ ] [Optional] When you input a long score number the UI breaks, please consider adding some kind of input validation to fix this

![Screenshot_20220518_180843](https://user-images.githubusercontent.com/25227022/169101900-9e2937f6-cdce-4ae5-9846-d22efa77ba2c.png)

Reply:
Thank you for the review and the recommendation.